modules/swagger-codegen/src/main/resources/csharp/compile.mustache

```diff
@@ -17,5 +17,9 @@ if not exist ".\bin" mkdir bin
 
 copy packages\Newtonsoft.Json.8.0.3\lib\{{targetFrameworkNuget}}\Newtonsoft.Json.dll bin\Newtonsoft.Json.dll
 copy packages\RestSharp.105.1.0\lib\{{targetFrameworkNuget}}\RestSharp.dll bin\RestSharp.dll
+copy packages\Fody.1.29.2\Fody.dll bin\Fody.dll
```

Comment:
@wing328 Ah yes, sorry missed that. Also can someone with the correct environment update the build.sh file? I don't have the env for it.